### PR TITLE
Add separate requests for comment tracker

### PR DESCRIPTION
**Pull request by pete-gov**

### CODE REVIEW

File: pages/updates/rfcs.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+    <td><b>Open Date</b></td>
+    <td><b>Close Date</b></td>
+  </tr>
+  <tr valign=top>
+    <td><b><a href="https://github.com/FedRAMP/rfc0001-new-comment-process/blob/main/rfc/0001.md">A New Comment Process for FedRAMP</a></b></td>
+    <td>Extended open discussion period for feedback on the pilot FedRAMP RFC process using GitHub.</td>
+    <td>2024-11-01</td>
+    <td>2024-12-31</td>
+  </tr>
+</table>
```
Comment (aj-stein-gsa) [inline pages/updates/rfcs.md:25]: Are we doing this because Markdown tables do not work so well with the current Jekyll table ot some other reason?

**Review by aj-stein-gsa — APPROVED** [review]
One question and recs on hyperlinks, but even without those suggestions, I approve.

File: pages/updates/rfcs.md (hunk 1 above)
Comment (pete-gov) [reply pages/updates/rfcs.md:25]: The old template the site uses seems to handle markdown tables like complete garbage - they don't word wrap or pad properly. At some point we'd be better off fixing whatever is going on with the CSS there but interim this is just easier.

File: pages/updates/rfcs.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+## Background
+
+44 U.S. Code § 3609 (a)(6) requires FedRAMP to _"establish and maintain a public
+comment process for proposed guidance and other FedRAMP directives that may have
+a direct impact on cloud service providers and agencies before the issuance of
+such guidance or other FedRAMP directives."_
+
+In the beginning of FY25, FedRAMP is piloting a Request for Comments (RFC)
+process using GitHub, with individual RFCs posted as separate repositories with
+Discussion features enabled. Stakeholders will be able to discuss and interact
```
[the remark this answers is not in the document]
Comment (pete-gov) [reply pages/updates/rfcs.md:36]: I was thinking at some point we should move/copy some of the more detailed explanation stuff from the actual RFC 0001 and the GitHub README for our RFCs over here - I guess we could just move all of that into the Background here?

[the remark this answers is not in the document]
Comment (aj-stein-gsa) [reply pages/updates/rfcs.md:36]: Either way, I'm fine. I just know my nerd talk is either very approachable to others or completely foreign. If we are going to address that elsewhere, also fine.

**Comment by pete-gov** [comment]
I'm going to actually cancel this pending some traps.

**Pull request closed without merging**